### ISSUE TITLE
Change folder to cache for changed user-agent file

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/firefox-ios#13754]
The file “changed-ua-set-of-hosts.xcarchive” is currently saved under documents. Let’s save this under the cache instead.

![image](https://user-images.githubusercontent.com/11338480/228022443-8ad9fedf-153c-4e7e-b2b4-7753b9cd99e9.png)

!image-20230327-173756.png|width=1252,height=724!

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXIOS-6074)

**Comment by chriswebb09** [comment]
@lmarceau is this one still available? The fix looks relatively straightforward.

**Comment by lmarceau** [comment]
Yes available, just note that we should migrate existing users to the new location as well for a couple of versions (I forgot to note that in the task).

**Comment by lmarceau** [comment]
This task is still up for grabs if anyone is interested!

**Comment by demo-lvrs** [comment]
Happy to give this one a try!

**Comment by demo-lvrs** [comment]
This task has been completed

**Comment by data-sync-user** [comment]
➤ Laurie Marceau commented:

For QA it can be tested the "changed-ua-set-of-hosts.xcarchive" doesn’t appear under documents (beside Downloads). this file should appear when we change user agent on certain websites (mobile VS desktop).

**Comment by data-sync-user** [comment]
➤ Andrei Bodea commented:

Verified as fixed on v144 (62506) with iPhone 15 Pro (18.6).

!image-20251008-121506.png|width=406,alt="image-20251008-121506.png"!
